arb_os/evmCallStack.mini

```diff
@@ -1306,6 +1313,19 @@ public impure func evmCallStack_oldestCallFrame() -> option<EvmCallFrame> {
     };
 }
 
+public func evmCallFrame_constructorIsOnStackForAddress(frame: EvmCallFrame, addr: address) -> bool {
```

Comment:
I'm wondering if there are any edge case deviations here involving nested CREATE2s

CREATE2 contract A
    CREATE2 contract A
    EXTCODESIZE contract A
    SELFDESTRUCT contract A

I think in both ethereum and arbitrum the operations would be valid, but with this implementation in arbitrum  the EXTCODESIZE would return 0 rather than the size of the code

Reply:
I haven't tested it, but I think that if there is a CREATE2 for address A, with another CREATE2 for the same address A nested inside it, the second/inner CREATE2 will fail because it will see that there is already a contract at A.

Reply:
That should be the case from what I remember of the code, as CREATE2 instantly populates contractInfo, but will fail if contractInfo already exists.